### PR TITLE
Update resources.md

### PR DESCRIPTION
**Pull request by keramix**
Added a link to the official releases for Windows.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@keramix : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/python/resources.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ A list of links that we recommend for Python Developers working on machines runn
 
 - [Python and Visual Studio Code](https://code.visualstudio.com/docs/languages/python): An overview of using Python with VS Code.
 
+- [Python Releases for Windows](https://www.afternerd.com/blog/round-number-nearest-integer/): The official Python releases for Windows.
```
Comment (mattwojo) [inline hub/python/resources.md]: afternerd.com? 
Link should be: https://www.python.org/downloads/windows/

**Comment by mattwojo** [comment]
Thanks for contributing. I fixed the link URL and am merging.

**Pull request merged**